### DISCUSSION TITLE
Motion detection on wireless doorbell

### DISCUSSION THREAD

**Discussion by PeterAlexander75** [samemory/homebridge-eufy-security-control#193] [Q&A] [answered]
Have updated the plug-in after not using for some time. Am using rc0 - motion detection and notifications from HomeKit seem to have stopped. They are working fine from the native app. Is this intended or a bug?

**Comment by lenoxys** [comment]
Weird I've done the test on my setup and I got the notification. Have you enabled push notification ? If yes, can you open an issue ?

https://github.com/samemory/homebridge-eufy-security/issues/new?assignees=&labels=bug&template=bug-report.yml&title=%5BBug%5D%3A+

**Comment by PeterAlexander75** [comment]
Ok and thanks. I’ll do some more testing and raise if it persists.

**Comment by PeterAlexander75** [comment, marked as answer]
So, I upgraded to rc3 this morning. "Livestream from camera device" was off by default (and has been since I upgraded from a much older version). With this off, in HomeKit the motion sensor was set to trigger notifications, but it didn't.

Turned on the "Livestream from camera device" setting and rebooted. Went into HomeKit and the Doorbell camera had notifications turned off. Turned them on. This doesn't seem to have made any difference, but streaming is more reliable for me.

I'll raise an issue.
